### PR TITLE
Feature/remove sum rows on reporting

### PR DESCRIPTION
**Pull request by yepzdk**
#### Link to ticket

https://leantime.itkdev.dk/tickets/showKanban?tab=timesheet#/tickets/showTicket/1814

#### Description

* Remove rows with same values. 
* Fix duration widget bug
* Remove summery on quick_entry_box

#### Screenshot of the result

Screenshot 2024-06-27 at 15 03 12
Screenshot 2024-06-27 at 14 24 03

#### Checklist

- [ ] My code is covered by test cases.
- [ ] My code passes our test (all our tests).
- [ ] My code passes our static analysis suite.
- [ ] My code passes our continuous integration process.

### CODE REVIEW

File: Resources/views/app/default/_form_modal.html.twig [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{% extends '@App/default/_form_modal.html.twig' %}
+
+{% block modal_title %}
+    {# Ugly hack to change the generic title on timesheet_create modal #}
+    {% if title == 'Create' or 'Opret' %}
```
Comment (rimi-itk) [inline Resources/views/app/default/_form_modal.html.twig]: Use https://twig.symfony.com/doc/3.x/templates.html#containment-operator here: 

```suggestion
    {% if title in ['Create', 'Opret'] %}
```

File: Resources/views/app/base.html.twig [hunk 2]
```diff
@@ -18,6 +18,8 @@
     {{ include('partials/navbar_user.html.twig') }}
 {% endblock %}
 
+{% block title %}AAK-tid – {{ block('page_title') }}{% endblock %}
```
Comment (rimi-itk) [inline Resources/views/app/base.html.twig:21]: If you just want to add a prefix to whatever the original `title` block renders, you can use https://twig.symfony.com/doc/3.x/functions/parent.html here:

```suggestion
{% block title %}AAK-tid – {{ parent() }}{% endblock %}
```

File: Resources/views/app/default/_form_modal.html.twig (hunk 1 above)
Comment (yepzdk) [reply Resources/views/app/default/_form_modal.html.twig]: Changed it. Thanks for the more clean solution.

File: Resources/views/app/base.html.twig (hunk 2 above)
Comment (yepzdk) [reply Resources/views/app/base.html.twig:21]: We want to replace the default prefix.

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**